conduct.md

```diff
@@ -65,14 +65,15 @@ Allegheny's honor code for all submitted work.
 
 * Acceptable Conduct (A):
   * Members shall
-    * Watch the course website and Slack workspace for changes to due dates.
-    * Turn assignments in on time.
-    * Complete their work in a timely manner that will not negatively affect other group members.
+    * Turn assignments in on the due date; the only exception to this rule is if the student is excused by the professor.
+    * Keep track of the due dates of the assignments.
+    * Be able to give reasons for missing deadlines.
+    * Group work shall be turned in on time, with all group members agreeing on when the work is finished and ready to be turned in.
 
 * Unacceptable Conduct (U):
   * Members shall not
-    * Be dishonest about late assignments.
-    * Bargain for more time to complete assignments.
+    * Turn in work past the due date and time given.
+    * Turn work in without notifying teammates.
```

Comment:
The deleted changes are from other groups. I think that they should be kept since they were agreed on by others.